app/src/test/java/ar/scacchipa/twittercloneapp/domain/ConsumableAuthUseCaseTest.kt

```diff
@@ -0,0 +1,22 @@
+package ar.scacchipa.twittercloneapp.domain
+
+import org.junit.Assert
+import org.junit.Test
+
+class ConsumableAuthUseCaseTest {
+    @Test
+    fun consumableAuthUseCaseShouldReturnArl() {
```

Comment:
error type in name of fun

Reply:
The new name is "consumableAuthUseCaseShouldReturnUrl()"